docs/src/main/asciidoc/rest-client-reactive.adoc

```diff
@@ -286,7 +287,7 @@ public class ExtensionsResource {
 
     public ExtensionsResource() {
         extensionsService = RestClientBuilder.newBuilder()
-            .baseUri("https://stage.code.quarkus.io/api")
+            .baseUri(URI.create("https://stage.code.quarkus.io/api"))
```

Comment:
I wonder if we should add a way to use a string instead. But maybe it's a bad idea.

Reply:
It sounds like a good idea to me, but the user would have to downcast to get it